src/styles/blocks/body.scss

```diff
@@ -0,0 +1,56 @@
+@import "../utils/variebles";
+
+body {
+  display: flex;
+  height: $height-container;
+  font-family: Arial, sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: ($width-day * 7) + ($between-gap * 6);
+  margin: auto;
+
+  .day {
+    position: relative;
+    top: 0;
+    transition: top 0.5s ease;
```

Comment:
you change `background-color` too, it should be listed in transition-property

Reply:
I don't understand, where exactly this code should be placed?? 

Please explain in more detail or show an example ...